lib/register.php

```diff
@@ -221,6 +221,13 @@ function gutenberg_get_edit_post_url( $post_id ) {
  * @return string Edit post link.
  */
 function gutenberg_filter_edit_post_link( $url, $post_id, $context ) {
+	$sendback = wp_get_referer();
+	if ( $sendback && (
+		 strpos( $sendback, 'post.php' ) !== false ||
```

Comment:
Should we be more specific about these referrers, in case `post.php` occurs as a suffix for a file, or query parameter, or otherwise? Maybe:

```php
0 === strpos( parse_url( admin_url( 'post.php' ), PHP_URL_PATH ), $sendback )
```